tests/config/pytest_software.yml

```diff
@@ -71,31 +71,31 @@ bismark_methylation_extractor:
   - software/bismark/genome_preparation/**
   - tests/software/bismark/methylation_extractor/**
 
+blast_blastn:
+  - software/blast/blastn/**
+  - tests/software/blast/blastn/**
+
 blast_makeblastdb:
   - software/blast/makeblastdb/**
   - tests/software/blast/makeblastdb/**
 
-blast_blastn:
```

Comment:
Can you please update to the latest master version (e.g. `git rebase master` to make these unrelated changes go away?
Apparently, there are even some merge conflicts here

Reply:
checked out single file from upstream/master, resolving the conflict